package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "spellcheck": "mdspell --en-gb --report --ignore-acronyms --ignore-numbers --no-suggestions 'content/**/*.md'",
     "broken-link-local": "blcl --recursive --ordered --exclude-external ./public/",
-    "broken-link-all": "blcl --recursive --ordered ./public/",
+    "//": "Github links below are excluded to prevent breaking the link checker for the Edit in Github urls, when no such page exists in master",
```

Comment:
Eurgh, does npm's JSON parser really not support comments?

Reply:
https://groups.google.com/forum/#!msg/nodejs/NmL7jdeuw0M/yTqI05DRQrIJ

Reply:
ah, i never saw this hack. I'd rather remove this comment